cabal2nix/src/Distribution/Nixpkgs/Haskell/FromCabal/PostProcess.hs

```diff
@@ -405,5 +405,14 @@ bustleOverrides = set (libraryDepends . pkgconfig . contains "system-glib = pkgs
                 . set (metaSection . license) (Known "lib.licenses.lgpl21Plus")
                 . set (metaSection . hydraPlatforms) Nothing
 
+tzOverrides :: Derivation -> Derivation
+tzOverrides =
+  -- The tz Haskell package uses zoneinfo files from the system tzdata package in its tests.
+  set phaseOverrides "preCheck = \"export TZDIR=${system-tzdata}/share/zoneinfo\";" .
```

Comment:
This should be unnecessary, actually, since the `tzata` setup hook in nixpkgs sets this variable nowadays. Can you try removing this?

If we can drop this, I wonder if having no  cabal2nix-override would be better, since `system-tzdata` needs an external override anyways and it is an arbitrarily chosen name, so probably more confusing than not having it!

Reply:
Great suggestion, I made this change in c2b01f6 and it looks like it works well!

Reply:
> Want me to just drop the override?

Yes, I think that is the best solution to this unfortunate situation!

Reply:
Alright, I've just completely dropped the override in 945e8ee.